src/DataTypes/Serializations/SerializationInfoTuple.cpp

```diff
@@ -70,13 +70,15 @@ void SerializationInfoTuple::add(const SerializationInfo & other)
 
 void SerializationInfoTuple::addDefaults(size_t length)
 {
+    SerializationInfo::addDefaults(length);
```

Comment:
What is this line for?

Reply:
Because now empty tuples can be serialized and it should contain the same amount of information as other columns.

Reply:
Doesn't t change serialization for existing tuples?